docs/pages/guides/webhooks.mdx

```diff
@@ -61,12 +81,12 @@ available for subscription, along with their schema.
 
 Events available for use include:
 
-- UserEntered/UserLeft
 - StorageUpdated
+- UserEntered/UserLeft
+- RoomCreated/RoomDeleted
 
 More events will come later such as:
 
-- RoomCreated/RoomDeleted
 - MaxConnectionsReached
```

Comment:
```suggestion
- `MaxConnectionsReached`
```

Reply:
Guillame had mentioned putting these in backticks since they're technically event types... so this is a "my bad" from before fix